### PR TITLE
Fix/forms

### PR DESCRIPTION
**Pull request by HuuTai33**

### CODE REVIEW

**Review by Jberque — APPROVED** [review]

**Comment by obeone** [comment]
For the records, this PR was fixing various bugs on forms

File: index.html [hunk 1]
```diff
@@ -18,6 +18,8 @@
 <body>
   <div id="app"></div>
   <script type="module" src="/src/main.ts"></script>
+  <script src="https://unpkg.com/vue@3"></script>
```
Comment (utarwyn) [inline index.html:21]: @HuuTai33 Why did you re-import _vue3_ and _hcaptcha/vue3-hcaptcha_ from unpkg? They seem to be already included in the final bundle.

This version of the vue3 also corresponds to a development version... it doesn't seem optimal to me. If you check the dev console on ecocode.io, the following message appears:
> You are running a development build of Vue.
> Make sure to use the production build (*.prod.js) when deploying for production.

Comment (HuuTai33) [reply index.html:21]: oh yes my bad, will fix it !

Comment (obeone) [reply index.html:21]: Fixed in #6

**Pull request merged**